environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - numpy
+  - numpy==1.23
```

Comment:
Why do you need to fix this?

Reply:
category_encoder only supported numpy versions lower than 1.23.

You can actually ignore my pull request. Since we can only use numeric columns in the initial dataset I actually don't need category_encoders. Sorry for the disturbance.